build/configs/artik053/grpc/defconfig

```diff
@@ -425,7 +425,7 @@ CONFIG_ADC=y
 CONFIG_ADC_FIFOSIZE=8
 CONFIG_ADC_NPOLLWAITERS=2
 # CONFIG_DAC is not set
-# CONFIG_DRIVERS_KERNEL_TEST is not set
+# CONFIG_DRIVERS_OS_API_TEST is not set
```

Comment:
This config is used in many Kconfig files. 
Please check and change in all these places. 

apps/examples/testcase/le_tc/drivers/Kconfig:	select DRIVERS_KERNEL_TEST
apps/examples/testcase/le_tc/compression/Kconfig:	select DRIVERS_KERNEL_TEST
apps/examples/testcase/le_tc/network/Kconfig:	select DRIVERS_KERNEL_TEST
apps/examples/testcase/le_tc/kernel/Kconfig:	select DRIVERS_KERNEL_TEST
apps/examples/rdp/Kconfig:	select DRIVERS_KERNEL_TEST
apps/examples/mpu/Kconfig:	select DRIVERS_KERNEL_TEST
apps/examples/stack_protection/Kconfig:	select DRIVERS_KERNEL_TEST